### ISSUE TITLE
div128to64 is broken

### ISSUE THREAD

**Issue by piquan** [piquan/CUDA-uint128#2]
I get a divide-by-zero on my laptop, and Nick sees the top half getting lost.

**Comment by piquan** [comment]
Test case:

```
#include <iostream>
#include <cmath>

#include <cuda_uint128.h>

using namespace std;

int
main()
{
    uint128_t dividend = 12345;
    uint64_t divisor = 5;
    uint64_t remainder;
    uint64_t quotient = div128to64(dividend, divisor, &remainder);
    cout << dividend << " / " << divisor << " = " 
         << quotient << " r " << remainder << endl;
}
```

**Comment by piquan** [comment]
Bug in clzll, fixed in 62b28d8d8a3bb6ed3608985dbf51605c8141064e
